presets/massa-station-preset.js

```diff
@@ -48,6 +52,8 @@ module.exports = {
         neutral: colorWhite,
         info: colorDarkGrey,
         success: colorBrand,
+        warning: colorWarning,
+        error: colorError,
```

Comment:
I don't see **warning** and **error** (at least named like that) in the presets colors. Could you, please, double check this names if we need it!?

Reply:
are you talking about the figma ? https://www.figma.com/file/LSaOibVDFb1reF6FePVn5l/Massastation?type=design&node-id=1-2120&t=BTSMnr173C8izK3a-0
these 2 colors were in the figma but not in the tailwind config
cc @maxime-carabina

Reply:
Yes, please sync with @maxime-carabina on that since he also needs. Also agree with Lucas about the color names and add in ONE place only. Then you can use it here.